src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -2760,13 +2760,12 @@ partial class BoundIsPatternExpression
         public override void Accept(OperationVisitor visitor)
         {
             // TODO: implement IOperation for pattern-matching constructs (https://github.com/dotnet/roslyn/issues/8699)
-            throw new NotImplementedException();
```

Comment:
Yes, once the "something useful" is defined for IOperation. That is why there is a reference to issue https://github.com/dotnet/roslyn/issues/8699 here.